bundles/org.openhab.binding.tesla/src/main/resources/OH-INF/i18n/tesla.properties

```diff
@@ -352,3 +358,8 @@ channel-type.tesla.wakeup.label = Wake Up
 channel-type.tesla.wakeup.description = Wake up the vehicle from a (deep) sleep
 channel-type.tesla.wiperbladeheater.label = Wiperblade Heater
 channel-type.tesla.wiperbladeheater.description = Indicates if the wiperblade heater is switched on
+
+# channel types
+
+channel-type.tesla.softwareversion.label = Software version
```

Comment:
Think the i18n files need to be regenerated.

Reply:
Done and pushed.

Reply:
What am i missing, i would not expect this as software version is no longer a channel, but a property as it was before.